HACKING.md

```diff
@@ -311,43 +322,61 @@ To make a release of the ops library, do the following:
    notes field.
-5. Group the changes by the commit type (feat, fix, etc.) and use full names (e.g., "Features",
-   not "feat") for group headings. Strip the commit type prefix from the bullet point. Strip the
-   username (who did each commit) if the author is a member of the Charm Tech team.
-6. Where appropriate, collapse multiple tightly related bullet points into a
+6. Group the changes first by package (`ops` and `ops-scenario`) and then by
+   the commit type (feat, fix, etc.) and use full names (e.g., "Features",
+   not "feat") for group headings. Strip the commit type prefix from the bullet point,
+   and capitalise the first word. Strip the username (who did each commit) if the
+   author is a member of the Charm Tech team.
+   For example: the PR `docs: clarify where StoredState is stored`
+   becomes `* Clarify where StoredState is stored` in the "Documentation" section.
+7. Where appropriate, collapse multiple tightly related bullet points into a
    single point that refers to multiple commits.
-7. Create a new branch, and copy this text to the [CHANGES.md](CHANGES.md) file,
+8. Create a new branch, and copy this text to the [CHANGES.md](CHANGES.md) file,
    stripping out links, who did each commit, the new contributor list, and the
    link to the full changelog.
-8. Change [version.py](ops/version.py)'s `version` to the
+9. Change [version.py](ops/version.py)'s `version` to the
    [appropriate string](https://semver.org/).
-9. Check if there's a `chore: update charm pins` auto-generated PR in the queue. If it looks
+10. Check if there's a `chore: update charm pins` auto-generated PR in the queue. If it looks
    good, merge it and check that tests still pass. If needed, you can re-trigger the
    `Update Charm Pins` workflow manually to ensure latest charms and ops get tested.
-10. Add, commit, and push, and open a PR to get the changelog and version bump
+11. Add, commit, and push, and open a PR to get the changelog and version bump
    into main (and get it merged).
-11. Back in the GitHub releases page, tweak the release notes - for example,
+12. Back in the GitHub releases page, tweak the release notes - for example,
    you might want to have a short paragraph at the intro on particularly
    noteworthy changes.
-12. Have someone else in the Charm-Tech team proofread the release notes.
-13. When you are ready, click "Publish". (If you are not ready, click "Save as Draft".)
-
-This will trigger an automatic build for the Python package and publish it to
-[PyPI](https://pypi.org/project/ops/)) (authorisation is handled via a
+13. Have someone else in the Charm-Tech team proofread the release notes.
+14. When you are ready, click "Publish". (If you are not ready, click "Save as Draft".)
+15. If the release includes an `ops-scenario` package, then create a new tag in
+   the form `scenario-<major>.<minor>.<patch>`.
+
+This will trigger automatic builds for the Python packages and publish them to
```

Comment:
Maybe we should be clearer here about what triggers the automatic builds -- tag creation, right? If we're releasing one of ops or scenario, then the sole tag is created on publishing the release. But if we're releasing both, then the ops tag is created on publishing a release (triggering an automatic build). While the second tag is going to be created after the release (triggering the second automatic build). Is that right?